src/main/java/org/apache/datasketches/theta/IntersectionImplR.java

```diff
@@ -82,15 +83,15 @@
     }
   }
 
-  IntersectionImplR(final short seedHash) {
-    seedHash_ = seedHash;
-    mem_ = null;
-    lgArrLongs_ = 0;
-    curCount_ = -1;
-    thetaLong_ = Long.MAX_VALUE;
-    empty_ = false;
-    hashTable_ = null;
-  }
+  //  IntersectionImplR(final short seedHash) {
+  //    seedHash_ = seedHash;
+  //    mem_ = null;
+  //    lgArrLongs_ = 0;
+  //    curCount_ = -1;
+  //    thetaLong_ = Long.MAX_VALUE;
+  //    empty_ = false;
+  //    hashTable_ = null;
+  //  }
```

Comment:
Did you mean to remove this constructor?

Reply:
Yes.  It was placed there at one time when we thought we needed it.  But nothing calls it, it is dead code.  I left it there in case I thought of another reason to put it back.